recipe/build.sh

```diff
@@ -1,30 +1,14 @@
 set -exou
 
-if [[ $(arch) == "aarch64" || $(uname) == "Darwin" ]]; then
-pushd qtwebengine-chromium
-
-# Ensure that Chromium is built using the correct sysroot in Mac
-awk 'NR==77{$0="    rebase_path(\"'$CONDA_BUILD_SYSROOT'\", root_build_dir),"}1' chromium/build/config/mac/BUILD.gn > chromium/build/config/mac/BUILD.gn.tmp
```

Comment:
I'm still not sure about what to do about this particular modification.

Reply:
Do you still want to try to restore this modification? I think it should be convertible into a regular patch now that we're not checking out from Git.

Reply:
i was actually coming here to delete these lines.